packages/lit-dev-content/site/docs/v2/components/events.md

```diff
@@ -168,6 +168,10 @@ When listening to events on repeated items, it's often convenient to use [event
 
 {% playground-example "docs/components/events/list/" "my-element.ts" %}
 
+### Removing event listeners
+
+Passing `null`, `undefined` or `nothing` to an `@` expression will cause any existing listener to be removed, or simply no-op if no listener has been added.
```

Comment:
I don't think the no-op part is really necessary

Reply:
```suggestion
Passing `null`, `undefined` or `nothing` to an `@` expression will cause any existing listener to be removed.
```

Reply:
This PR was specifically addressing a user question that was asked in a forum: "what's a value I can pass to an event part to do nothing (aka be fast)?" Saying only what values remove an event listener still requires the user to infer whether passing undefined is a good default value to avoid extra cost.